application-chartjs-ui/src/main/resources/XWiki/ChartJS/WebHome.xml

```diff
@@ -336,6 +337,14 @@ require(['jquery', 'chart', 'datalabels', 'palette'], function($, Chart, datalab
         data: data,
         options: options
       });
+
+      // If the table is hidden, move it to the canvas element and show it, for accessibility purpose.
+      var table = $('table#' + data.table);
+      if (table.hasClass('hidden')) {
+        table.appendTo(canvas);
+        table.removeClass('hidden');
```

Comment:
That's because the screen readers take into account the CSS?

Reply:
Indeed, screen readers won't read elements that are marked as hidden. I added a comment in the latest commit.